src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -23,6 +23,9 @@
     <Compile Include="$(CommonTestPath)\System\TypeBuilderExtensions.cs" />
   </ItemGroup>
   <ItemGroup>
+      <CodeCoverageAssemblies Include="Microsoft.CSharp" /> 
```

Comment:
I have a PR to add an `AdditionalCodeCoverageAssemblies` item that could be used here. Don't feel the need to update the PR just for that, though; I can clean it up afterwards if necessary.

https://github.com/dotnet/corefx/pull/15441